### PR TITLE
BETA: Register laman .is-a.dev

### PR DESCRIPTION
**Pull request by Derogater**
Added `laman .is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by Derogater** [comment]
Testing a dev website.

File: domains/laman .json
Comment (DEV-DIBSTER) [inline domains/laman .json:1]: No space allowed in domain name.

File: domains/laman .json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Derogater",
+        "email": "thelazybloggers@gmail.com"
+    },
+    "record": {
+        "TXT": "laman"
```
Comment (DEV-DIBSTER) [inline domains/laman .json:7]: TXT record alone doesn't do anything.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/laman .json (hunk 1 above)
Comment (Derogater) [reply domains/laman .json:7]: What should be done then?

**Pull request closed without merging**